plugins/modules/credential.py

```diff
@@ -201,6 +220,19 @@ def main() -> None:
         else (controller.get_item_name(credential) if credential else name)
     )
 
+    try:
```

Comment:
I think this should not fail when the user does not want to copy anything so copy_from is not set. If I understand correctly, it will fail right now if I do not set copy_from.

Reply:
Updated to check for copy_from param, before we get into this try block.